public/app/pipeline-orchestrator.js

```
@@ -779,7 +856,7 @@ class PipelineOrchestrator {
     orch._preWarmWorklet = (async () => {
       try {
         const suspCtx = new (window.AudioContext || window.webkitAudioContext)({ sampleRate: 48000 });
```

ESLint

[error] 'window' is not defined.

(no-undef)